static/sass/styles.scss

```diff
@@ -313,3 +313,9 @@ sup {
     margin-bottom: 1rem;
   }
 }
+
+/// XXX Give card headers an underlined option
+/// This will need replacing when the latest vanilla card changes land
+.p-card__header--bordered {
```

Comment:
I think that I'll clean this up better. The problem is existing card headers on the site not having borders.